issues-guide.md

```diff
@@ -0,0 +1,100 @@
+are handled and processed. 
+
+
+The Sprint
+-----------
+We are on a weekly sprint cycle and every Monday we get together for a meeting
+on Google Hangouts. Currently our meetings are at 12pm PDT / 7pm UTC and the 
+link is always
+posted to the IRC channel. This is where collaborators are assigned tasks and 
+issues to work on for the week. Note that anyone can become a collaborator and 
+all they have to do is contact
+[@jbenet](https://github.com/jbenet) and join in on a sprint meeting. On the 
+rest of the days when we aren't meeting on 
+Google Hangouts we have a "standup" at the same time in the IRC channel. This is
+a time where everyone is on the channel at once so we can sync up and bring up 
+any topics that need to be discussed. 
+
+
+Github Labels
+--------------
+We make extensive use of Github labels to help us and our tools organize issues.
+Here is a rough guide.
+
+__The colors:__  
+* blue: topic / subsystem  
+* light red (important topic): security things  
+* red (very important): bugs!!  
+* orange (important): cleanup / refactor things  
+* light green: the difficulty level  
+* dark green: testing  
+* yellow: documentation  
+* gray labels: used by tools like [waffle.io](https://waffle.io/ipfs/ipfs)  
+
+__Naming:__  
+It's very ad-hoc with no strict science. Generally, the names should be
+understandable by themselves. For example, use "difficulty: medium" instead
+of "medium".
+
+
+Waffle
+-------
+[Waffle](https://waffle.io/ipfs/ipfs) is our issue board and is divided into 
+7 columns:
+* New Issues: This is pretty self explanatory and is where all the new issues
+    get thrown into.
+* Icebox: These are issues we've decided that aren't important for now, but
+    will want to get to some day.
+* Backlog: These are the issues that are ready for work to begin on and we 
+    will pull from these when determining what to work on for 
+    the next sprint cycle.
+* Sprint: These are the issues that are being tackled for the current sprint.
+* In Progress: These are the issues that people are working on at the current
+    point in time.
+* Code Review & QA: These are the issues where work has been completed for 
+    and are awaiting approval to be merged or closed.
+* Done: All the issues that have been closed in the past week.
+
+
+Issue Workflow Example
+-----------------------
+### New issue submitted
+1. The issue is submitted:
+    * The issue gets sent to the New Issues column in waffle.
+    * A collaborator on the repo intercepts the issue and begins discussion on
+      the issue.
+2. The issue has been sufficiently described for work to begin:
+   * If work should begin soon on the issue then a collaborator moves the issue
+     to the backlog column, else if the issue should be held off from work for
+     a later time then it is moved to the Icebox column.
+3. In a sprint meeting, a collaborator decides to work on an issue:
+    * The issue is moved to the Sprint column and assigned an owner.
```

Comment:
What if non-collaborators are working on an issue?  It seems like you need to have [write access to the relevant repository][1] before you can adjust issue labels, and Waffle needs that for shifting stuff between columns.

[1]: https://help.github.com/articles/permission-levels-for-an-organization-repository/